### ISSUE TITLE
Getting an error when trying to use price property

### ISSUE THREAD

**Issue by ericuldall** [jofftiquez/vue-stripe-checkout#112]
Trying to check out with a single payment as described here: https://stripe.com/docs/payments/checkout/client

This library works fine with subscriptions using {plan} but when I try using {price} i get the following error:
```
Invalid stripe.redirectToCheckout parameter: items.0.price is not an accepted parameter.
```

I notice in the stripe checkout docs there is an option `mode = payment|subscription` and I don't see that implemented in the code here. Does this library support one time payments?

**Comment by pilotkid** [comment]
@jofftiquez any update on this?

**Comment by jofftiquez** [comment]
Hello, thanks for this. I will look into this soon. My apologies, I've been very busy with work lately. I hope you understand. A PR is always welcome.

**Comment by watsuyo** [comment]
I'm facing the same problem.

**Comment by ivgtr** [comment]
Hey @watsuyo 
This issue should be resolved in #118.
Or clone this repository and add the lineItems and items properties.

**Comment by jofftiquez** [comment]
Hello please check the latest version. There's a significant improvements to the properties. 

Thanks @ivgtr
